qa-core/src/account-api/api/apis/AuthAPI.ts

```diff
@@ -27,4 +27,24 @@ export default class AuthAPI extends BaseAPI {
       return [res, cookie]
     }, opts)
   }
+
+  async getSsoProvider(
```

Comment:
I think it would be nice to move the functions that aren't directly related to license key activation out to another branch, things like this usually help with the review or if someone looks back at the git history of a file.